### PR TITLE
enable activeMQ build

### PR DESCRIPTION
**Pull request by afsanjar**

### CODE REVIEW

**Review by gerrith3 — CHANGES_REQUESTED** [review]
Okay - would would it take to base this on ubi8 instead of centos:7?  Since 7 is old and centos is kind of going away, this would future proof us for a while.

I didn't see a LICENCE file - each build script should be licensed under Apache 2.0.

I defer to Priya for any other comments.

File: a/activemq/Dockerfiles/latest_centos_7/build/Dockerfile
Comment (seth-priya) [inline a/activemq/Dockerfiles/latest_centos_7/build/Dockerfile:1]: agree with Gerrit on using ubi 8 instead of centos:7 unless of course there is an explicit need for centos:7 and I believe the image should come with a multiarch manifest, so we may not need the "ppc64le/" part explicitly

File: a/activemq/Dockerfiles/latest_centos_7/build/build.sh
Comment (seth-priya) [inline a/activemq/Dockerfiles/latest_centos_7/build/build.sh:1]: we generally keep the build scripts in the top level package folder which would be https://github.com/ppc64le/build-scripts/tree/master/a/activemq in this case and also have a specific header for each script like https://github.com/ppc64le/build-scripts/blob/master/a/aaargh/aaargh_rhel_7.3.sh#L1 to https://github.com/ppc64le/build-scripts/blob/master/a/aaargh/aaargh_rhel_7.3.sh#L16 which lists down the package, distro version, Maintainer name etc. which makes them consistent and easier to share with other teams etc. The header also has the script license which we set as "Apache 2.0"

File: a/activemq/Dockerfiles/latest_centos_7/run/Dockerfile
Comment (seth-priya) [inline a/activemq/Dockerfiles/latest_centos_7/run/Dockerfile:1]: same comment as earlier about use of centos:7 and "ppc64le/"

**Review by seth-priya — CHANGES_REQUESTED** [review]

File: a/activemq/Dockerfiles/latest_centos_7/build/Dockerfile
Comment (afsanjar) [reply a/activemq/Dockerfiles/latest_centos_7/build/Dockerfile:1]: Regarding selecting centos7, I agree with @gerrith3 and @seth-priya. However, ActiveMQ builds/Run requires java-7 https://activemq.apache.org/version-5-getting-started.html#installation-procedure-for-unix. I'll try ubi8, not sure if it supports JAVA 7.

Comment (afsanjar) [reply a/activemq/Dockerfiles/latest_centos_7/build/Dockerfile:1]: regarding the license file, there was already a one at the root https://github.com/ppc64le/build-scripts/tree/master/a/activemq

File: a/activemq/Dockerfiles/latest_centos_7/run/Dockerfile
Comment (afsanjar) [reply a/activemq/Dockerfiles/latest_centos_7/run/Dockerfile:1]: okay

File: a/activemq/Dockerfiles/latest_centos_7/build/build.sh
Comment (afsanjar) [reply a/activemq/Dockerfiles/latest_centos_7/build/build.sh:1]: Thanks, will do

**Review by gerrith3 — APPROVED** [review]
Well, that looks much better, and much simpler to me!

The only minor thing that I would love to be able to add is the ability to specify the Version on the command line - by default build 5.17.1 but if someone wants to try another version, it is simply an argument, e.g. 5.17.2 and following.  Might break for 5.18 or not, but then we can get a new issue raised.  However, that is a  "would be nice" - not sure if Priya has a boilerplate for that, since we probably have only used it occasionally.

Oh, and is there a plan to publish this to dockerhub, quay,io, icr, gcr, red hat marketplace, etc.?  Since this is a commonly used dependency it would be nice to get it published at least one high visible place with appropriate tags.

Letting Priya do a final review/approval (or change request) but I'm good with it as it stands.

**Comment by seth-priya** [comment]
yes, we have not been very consistent with the version bit, but here is a recent example https://github.com/ppc64le/build-scripts/blob/master/j/JTransforms/JTransforms_ubuntu_16.04.sh#L22
essentially just have a default one and give the user to enter his own. But we should probably do more checking as to the validity etc, else it will just fallback to master

Approving this for now

**Review by seth-priya — APPROVED** [review]

**Pull request merged**
